src/main/cms/cms_menu_rpm_limit.c

```diff
@@ -70,6 +73,7 @@ static const OSD_Entry cmsx_menuRpmLimitEntries[] =
     {  "ACTIVE",   OME_Bool | REBOOT_REQUIRED,  NULL, &rpm_limit },
     { "MAX RPM", OME_UINT16, NULL, &(OSD_UINT16_t){ &rpm_limit_value, 0, UINT16_MAX, 100} },
     { "KV", OME_UINT16, NULL, &(OSD_UINT16_t){ &kv, 0, UINT16_MAX, 1} },
+    { "RPM FILTER CUTOFF", OME_UINT16, NULL, &(OSD_UINT16_t){ &rpm_limit_rpm_filter_cutoff, 1, UINT16_MAX, 1000} }, 
```

Comment:
```suggestion
    { "LOWPASS CUTOFF", OME_UINT16, NULL, &(OSD_UINT16_t){ &rpm_limit_rpm_filter_cutoff, 1, UINT16_MAX, 1000} }, 
```

Reply:
Same concern as above. I agree we don't want to confuse this filter with the gyro RPM filter, but I think we still need to distinguish from other filters used by the RPM limiter. Maybe "AVG RPM CUTOFF"?